deployments/manifest.yml

```
@@ -495,18 +487,42 @@ spec:
+  name: sandbox-instances
+  namespace: browser-sandbox
+spec:
+  clusterIP: None # Makes the service headless
   selector:
-    app: browser-sandbox-frontend
+    app: browser-sandbox-test # Selects pods with label app=browser-sandbox
+  ports:
+    - name: rdp
+      port: 3389
+      targetPort: rdp # Matches the named port "rdp" (3389) in the pod spec
+---
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)